search/views.py

```diff
@@ -310,7 +318,6 @@ def auto_suggestion(request):
         search_term = request.POST.get("search_string", None)
         querystring = {
             "q": search_term,
-            "is_archived": "false",
```

Comment:
If we leave this param here, the structure of the responses will be changed (haystack behaviors), The best practice, in this case, is to check is_archived of the response.